content/post/always-as-a-design-problem.md

```diff
@@ -0,0 +1,104 @@
++++
+title = "It's always a design problem"
+description = "Why you can't blame the player for making your game unfun"
+tags = [
+	"game-design",
+	"fonts-of-power"
+]
+date = 2021-03-30 #TODO: Update date
+author = "Alice I. Cecile & Rose Peck"
+draft = true
++++
+
+As a designer, watching players actually experience your game (or application!) for the first time can be an excruciating experience.
+They'll miss the "obvious" approach to a task, use bizarre workarounds, skip all of your attempts to teach them and then complain that nothing works!
+
+Faced with this, it's tempting to blame the foolish players for failing to grasp your vision: calling them lazy or dense and exclaiming that no, it's the kids who are wrong!
+
+But, not only is this morally questionable, it's a useless dead-end.
+Think like this and you'll often miss both important design flaws and clues from players about how to fix them.
+Try as you might, you will not be able to build a better gamer.
+You can curate your audience to some degree, but, by-and-large you're stuck with them.
+
+Instead, understand that it's **always a design problem**.
+Once you accept this bitter truth, you can start to make games that players understand naturally.
+At its heart: make it so **the easy thing to do is the right one**.
+
+## The Consumables Problem
+
+Let's make this a little more concrete by talking about a common case and how we solved it in our RPG, *Fonts of Power*.
+**Consumables** are a staple of the RPG genre, and see widespread use outside of that.
+In almost every game that they're used however, you see the same pattern play out again and again:
+Most players (especially casual players) will horde consumables until the end of time, never using them even when faced with a serious challenge.
+At best, you may see them used all at once, completely trivializing a plot-critical climatic boss battle as they guzzle potions and drown the charismatic but creepy vampire in holy water.
+
+As mentioned, it's tempting (and often quite common!) to blame this pattern on the players.
+To say that they are too risk-averse, too stingy, or too afraid of a fair fight.
+However this misses the core problem that your players are responding to.
+Most consumable systems **fail to communicate their design** to the players.
+Players are left wondering:
+
+- What challenges they'll face, and how hard this particular challenge is
+- How much a given consumable will affect the outcome of a fight
+- When they'll be able to get more consumables
+- How many consumables they can sustainably use each fight
+- Whether this is the best time to use a particular consumable
+
+Players don't have interesting or meaningful choices, and so they become overly conservative or act randomly.
+
+It's not the player's fault that the game didn't balance and communicate what the trade-offs are for using a consumable.
+As we said, it's always a design problem.
+
+## Solving Consumables
+
```

Comment:
Now that we've recognized the true _design problem_ at the heart of consumables, we can actually solve the underlying issue.